### PR TITLE
Makes nice error message when Chrome isn't launched

### LINKED ISSUE #107: CLI should provide a human error message when unable to connect to CRI
Perhaps ask the user if they have launched chrome via our helper script.

### PR DESCRIPTION
**Pull request by paullewis**
Fixes #107

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -49,6 +49,13 @@ lighthouse({
   flags: cli.flags
 }).then(results => {
   Printer[cli.flags.json ? 'json' : 'prettyPrint'](log, console, url, results);
+}).catch(err => {
+  if (err.code === 'ECONNREFUSED') {
+    console.error('Unable to connect to Chrome. Have you run ./launch-chrome.sh?');
+  } else {
+    console.error('Runtime error encountered:', err);
+    console.error(err.stack);
+  }
```
Comment (sindresorhus) [inline cli.js]: Since this is an error, you should also set the exit code:

```js
process.exitCode = 1;
```

Comment (samccone) [inline cli.js]: :+1:

Comment (samccone) [inline cli.js]: I think process.exit(1) would also set the exit status code to 1 as well. :)

**Comment by paullewis** [comment]
Thx for the feedback!

**Comment by paulirish** [comment]
lgtm

**Comment by brendankenny** [comment]
LGTM#2

**Pull request merged**